packages/eslint-config/package.json

```diff
@@ -44,7 +44,7 @@
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
     "eslint-webpack-plugin": "3.2.0",
-    "prettier": "2.8.8",
+    "prettier": "3.0.0",
```

Comment:
@gmfvpereira what do you think?

Reply:
It sounds reasonable, but it might require running `prettier write` into nitro-web to prevent CI from failing. I'll add it to my todo list here to spin up a nitro PR that points to this PR here (I guess I've done that in the past, I just need to remember how to do so 😅)

Reply:
> It sounds reasonable, but it might require running `prettier write` into nitro-web to prevent CI from failing. I'll add it to my todo list here to spin up a nitro PR that points to this PR here (I guess I've done that in the past, I just need to remember how to do so 😅)

I'm doing that rn